oracle/config/api.go

```diff
@@ -33,10 +33,20 @@ type APIConfig struct {
 	// URL is the URL that is used to fetch data from the API.
 	URL string `json:"url"`
 
+	// Endpoints is a list of endpoints that the provider can query.
+	Endpoints []Endpoint `json:"endpoints"`
+
```

Comment:
can't this only be `Endpoints` and the minimum length just has to be 1?

can we remove URL?

Its unclear looking at the structure of the config what differentiates these two fields

Reply:
I think the idea is to introduce the ability to specify multiple endpoints in a non-breaking way.